src/Silk.Data/MediatR/Guilds/Config/UpdateGuildConfig.cs

```diff
@@ -37,35 +36,28 @@ public record Request(Snowflake GuildID) : IRequest<GuildConfigEntity>
         public Optional<List<ExemptionEntity>>           Exemptions           { get; init; }
         public Optional<List<InfractionStepEntity>>      InfractionSteps      { get; init; }
         public Dictionary<string, InfractionStepEntity>? NamedInfractionSteps { get; init; }
+        public bool                                      ShouldCommit         { get; init; } = true;
```

Comment:
I *prefer* to have properties ordered by length (see above), but that's just a nitpick.